specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -0,0 +1,3031 @@
+  ],
+  "security": [
+    {
+      "AzureKey": []
+    }
+  ],
+  "securityDefinitions": {
+    "AzureKey": {
+      "type": "apiKey",
+      "description": "The secret key for your Azure Cognitive Services subscription.",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "tags": [
+    {
+      "name": "RadiologyInsights"
+    }
+  ],
+  "paths": {
+    "/radiology-insights/jobs/{id}": {
+      "get": {
+        "operationId": "RadiologyInsights_GetJob",
+        "tags": [
+          "RadiologyInsights"
+        ],
+        "summary": "Get Radiology Insights job details",
+        "description": "Gets the status and details of the Radiology Insights job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique ID of the job.",
+            "required": true,
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 36,
+            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-_]+[a-zA-Z0-9]$"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.ExpandQueryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/RadiologyInsightsJob"
+            },
+            "headers": {
+              "Retry-After": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "An opaque, globally-unique, server-generated string identifier for the request."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/HealthInsightsErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "An opaque, globally-unique, server-generated string identifier for the request."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the status and details of the Radiology Insights job.": {
+            "$ref": "./examples/SuccessfulRadiologyInsightsResponse.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "RadiologyInsights_CreateJob",
+        "tags": [
```

Comment:
previous api-version, this is a POST
https://github.com/Azure/azure-rest-api-specs/blob/d6e6999f1230ca87fa0355d6d0735ab8b419fb67/specification/ai/data-plane/HealthInsights/preview/2023-09-01-preview/openapi.json#L442-L444